swap/htlc.go

```diff
@@ -151,6 +200,11 @@ func NewHtlc(version ScriptVersion, cltvExpiry int32,
 
 	switch outputType {
 	case HtlcNP2WSH:
+		p2wshPkScript, err := input.WitnessScriptHash(htlc.Script())
```

Comment:
nit: there's the same block at line 193, I guess the refactor commit that has been removed is now partially added but not completely.